tests/e2e/class.js

```diff
@@ -0,0 +1,71 @@
+import {expect} from 'chai'
+import Server from '../../src'
+import {getRandomString, createTestInstance, deleteTestInstance} from '../utils'
+
+global.META = {
+  socket: 'test-socket'
+}
+
+describe('Class', function () {
+  let _class = null
+  const testClassName = getRandomString()
+  const instanceName = getRandomString()
+
+  before(function (done) {
+    createTestInstance(instanceName)
+      .then(instanceObj => {
+        process.env.SYNCANO_INSTANCE_NAME = instanceObj.name
+        process.env.SYNCANO_API_KEY = process.env.E2E_ACCOUNT_KEY
+        _class = new Server()._class
+        done()
+      })
+      .catch(err => {
+        console.log(err)
+        err.response.text()
+          .then(text => {
+            console.log(text)
+            done(err)
+          })
+      })
+  })
+
+  after(function (done) {
+    deleteTestInstance(instanceName)
+      .then(() => {
```

Comment:
I added ticket to the 1.0 project https://github.com/Syncano/syncano-server-js/projects/1#card-2807179